weed/server/filer_grpc_server_sub_meta.go

```diff
@@ -226,6 +226,9 @@ func (fs *FilerServer) eachEventNotificationFn(req *filer_pb.SubscribeMetadataRe
 		} else if matchByDirectory(dirPath, req.Directories) {
 			// good
 		} else {
+			if req.PathPrefix == "" {
```

Comment:
this revert the logic when pathprefix is empty

Reply:
I have thought about it, if I understand correctly, in seaweed, all events will have a directory. The first character is `/`, so it should not have any effect; I also thought about using a pointer type, but it does not matter. There will be more changes in comparison